playbooks/roles/sanity-checker/files/sanity_check.py

```diff
@@ -98,7 +98,10 @@ def get_dev_mounts():
     for line in proc_mounts.split("\n"):
         parts = line.split()
         mount = Mount(parts[1], parts[0])
-        if mount.device.startswith("/dev"):
+        if (
+                mount.device.startswith("/dev")
+                or mount.mount.startswith("/var/")      # special cases, but also used by the system
```

Comment:
@eLRuLL could you update the docstring for this method to note this case, and also add a comment about why the /var/ mount device name doesn't start with /dev?

Reply:
@swalladge I think the docstring is still correct, this is not for a "device" with that name, it's a check to confirm it's mounted on system folder. I added more explanation in the comment.